technical-reports/format/design-token.md

```diff
@@ -107,17 +109,35 @@ The **description** property must be a plain JSON string, for example:
 
 ## Type
 
-Declares the type of the token. [See “Types”](#types) for more information.
+Declares the type of the token. [See supported "Types"](#types) for more information.
 
```

Comment:
Maybe we should add a bit more info to make it clear that while the `type` property is optional, every token does have an unambiguous type.

Perhaps something like this:

```suggestion

Design tokens always have an unambiguous type, so that tools can reliably interpret their value. If a token's type is not explicitly specified via the `type` property, then the token's type MUST be determined as follows:

* If the token's value is a reference, then its type is the type of the token being referenced.
* Otherwise, if any of the token's parent groups have a `type` property, then the token's type is inherited from the closest parent group with a `type` property.
* Otherwise, the token's type whichever of the basic JSON types (`string`, `number`, `boolean`, `object`, `array` or `null`) its value is.

```

Reply:
I've updated the PR to include your suggestion @c1rrus.